### ISSUE TITLE
Exploration of benzylic amines

### ISSUE THREAD

**Issue by david1597** [OpenSourceMalaria/Series4#50]
Reposting [this ](https://github.com/OpenSourceMalaria/Series4/issues/44#issuecomment-390650414) comment from @MedChemProf as it deserves it's own issue. 

> While @david1597 is working out the conditions for what looks to be an efficient route to new benzylic alcohol analogs, we were interested in synthesizing the amino variant through the traditional route. To be consistent, we will also make the tertiary alcohol as a comparison:

![image](https://user-images.githubusercontent.com/30584472/40695655-825e59ea-6405-11e8-877e-3e2f7a735b72.png)
![image](https://user-images.githubusercontent.com/30584472/40695668-8e56bc38-6405-11e8-855e-450fbf1ad81a.png)

**Comment by david1597** [comment]
I think this is a great idea and a timely one to pursue in terms of our current priorities. I saw this [paper](https://www.nature.com/articles/s41557-018-0020-0) yesterday and thought it could be a neat idea for our compounds  - and then remembered @MedChemProf had already suggested exploring an amino replacement for the benzylic alcohols. We're still waiting on confirmation but the Pfizer compound currently suggest that exploring this area is not only giving us great potency but also improving parameters such as lipophilic efficiency. I feel that more resources directed in this area, at this current moment in time, is a good thing.

**Comment by mattodd** [comment]
Agreed. I like the idea a lot. Non-trivial chemistry, so of particular interest to the organic chemists in OSM, but absolutely a justified target based on the work of the Pfizer team, I'd say, building on top of some of the previous SAR. If no naysayers, then we can eventually close this issue and install the compound on the [DCNYS](https://github.com/OpenSourceMalaria/Series4/wiki/Desirable-Compounds-Not-Yet-Synthesised) page with a back-link here.
